tutorials/notebooks/keras/ptq/example_keras_activation_threshold.ipynb

```diff
@@ -0,0 +1,866 @@
+        "Quantisation thresholds are used to map a distribution of 32bit float values to their 8bit quantised counterparts. Doing this with the least loss of data while maintaining the most representative range is important for final model accuracy.\n",
+        "\n",
+        "\n",
+        "MCT's Post-training quantisation uses a represenative dataset to evaluate a list of typical output activation values. The challenge comes with how best to match these values to their quantised counterparts. This process comprises of two main steps, zscore threshold, quantisation threshold. Initially anomolus values must be removed these values may result in the final threshold being greater than it needs to be (reducing granularity of mapping and there for also reducing accuracy of model).\n",
+        "Here MCT uses z-score thresholding on the activation values establishing a threshold value to remove anomolus values. The process by which this is used will be covered in another tutorial.\n",
+        "\n",
+        "Quantisation Threshold MCT has a number of error metrics for finding the best quantisation threshold. However, Mean squared error is typically the best performing and used by default.\n",
+        "\n",
+        "The error is calculated based on the difference between the float and quantised distribution. The threshold is sellected based on the minimum error. For the case of MSE;\n",
+        "\n",
+        "$$\n",
+        "ERR(t) = \\frac{1}{n_s} \\sum_{X \\in Fl(D)} (Q(X, t, n_b) - X)^2\n",
+        "$$\n",
+        "\n",
+        "$ERR(t)$ : The quantization error function dependent on threshold t.\n",
+        "ns: The size of the representative dataset, indicating normalization over the dataset's size.\n",
+        "\n",
+        "$\\sum$: Summation over all elements X in the flattened dataset $Fl(D)$.\n",
+        "\n",
+        "$F_l(D)$: The collection of activation tensors in the l-th layer, representing the dataset D flattened for processing.\n",
+        "\n",
+        "$Q(X, t, n_b)$: The quantized approximation of X, given a threshold t and bit width nb.\n",
+        "\n",
+        "$X$: The original activation tensor before quantization.\n",
+        "\n",
+        "$t$: The quantization threshold, a critical parameter for controlling the quantization process.\n",
+        "\n",
+        "$n_b$: The number of bits used in the quantization process, affecting the model's precision and size.\n",
+        "\n",
+        "To increase efficiency in calculating the threshold, the search space for best threshold is restricted to **Power of Two** values only. This both restricts number of potential values to a reasonable number and increases hardware efficency.\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "9c0e9543-d356-412f-acf1-c2ecad553e06",
+      "metadata": {
+        "id": "9c0e9543-d356-412f-acf1-c2ecad553e06"
+      },
+      "source": [
+        "Error methods supported by MCT;\n",
```

Comment:
i think listing the methods makes more sense joined to the equation section. will leave as is for now unless you feel strongly about it

Reply:
Listing the methods is great, I just meant that you give it a proper highlight because it's getting lost in the long introduction to the section.
Maybe just make the sentence: "Error methods supported by MCT" the title of a subsection